mission_magage:
    sort_basket_rois: [[[400, 130], [480, 170]],  # 1号框t/br
                       [[120, 140], [160, 200]],  # 2号框t/br
                       [[0, 200],   [50, 320]],   # 3号框t/br
                       [[500, 260], [640, 380]]]  # 4号框t/br
    go_to_grab:  # 前往采摘的途经点
        wp_num: 3
        #     [x,     y,     yaw, id, exit_l, exit_a, cnt]  [m, m, °, id, m, °, cnt]
        wp_1: [0.550, 2.520, 180, 21, 0.1,    5,      1]
        wp_2: [0.450, 1.750, 180, 23, 0.1,    5,      1]
        wp_3: [1.120, 1.750, 180, 23, 0.1,    5,      1]

    transit_pt:  # 中转点  避免碰撞
        wp_1: [1.820, 1.750,   0, 10, 0.05,   5,      2]
        
    grab:  # 每棵果树前的抓取点
        wp_num: 3
        #     [x,     y,     yaw, id, exit_l, exit_a, cnt]  [m, m, °, id, m, °, cnt]
        wp_1: [1.840, 1.280, 0.0, 9,  0.015,  2,      5]
        wp_2: [1.840, 0.840, 0.0, 8,  0.015,  2,      5]
        wp_3: [1.840, 0.438, 0.0, 7,  0.015,  2,      5]

    put:  # 每个放置框前的放置点
        wp_num: 4
        #     [x,     y,     yaw, id, exit_l, exit_a, cnt]  [m, m, °, id, m, °, cnt]
        wp_1: [1.910, 2.390, 0,   11, 0.02,   5,      2]
        wp_2: [1.940, 2.900, 90,  15, 0.02,   5,      2]
        wp_3: [1.390, 2.900, 90,  16, 0.02,   5,      2]
        wp_4: [1.490, 2.430, 90,  16, 0.02,   5,      2]

    go_back:  # 返回点
        wp_1: [1.250, 1.750, 0,   23, 0.1,    5,      1]

    park:  # 泊车点
        wp_num: 4
        #     [x,     y,     yaw, id, exit_l, exit_a, cnt]  [m, m, °, id, m, °, cnt]
        wp_1: [0.340, 2.190, 180, 22, 0.015,  2,      2]
        wp_2: [0.340, 1.740, 180, 23, 0.015,  2,      2]
        wp_3: [0.340, 1.300, 180, 24, 0.015,  2,      2]
        wp_4: [0.340, 0.860, 180, 25, 0.015,  2,      2]

grab_param:
    x_j5_2_claw: 0.085   # m
    z_j5_2_claw: 0.0115  # m

    sight_middle_h:         300         # pix
    sight_middle_w:         [269, 457]  # pix
    x_obj_to_cam_close:     0.20  # m
    x_obj_to_cam_middle:    0.245  # m
    x_obj_to_cam_far:       0.34  # m
